nixos/tests/keter.nix

```diff
@@ -0,0 +1,32 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "keter";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ jappie ];
+  };
+
+  nodes.machine = { config, pkgs, ... }: {
+    services.keter = {
+      enable = true;
+      keterPackage = pkgs.haskellPackages.keter;
+        bundle = {
+            domain = "example.com";
+            package = pkgs.writeShellScriptBin "run" ''
+              #! /bin/sh
+              ${pkgs.python3}/bin/python -m http.server $PORT
+            '';
+            executable = "run";
```

Comment:
Will need to be changed according to one of my earlier comments, a little reminder.

Reply:
yes this is a derivation now.